scanpy/experimental/pp/_highly_variable_genes.py

```diff
@@ -27,6 +28,86 @@
 )
 
 
+@nb.njit(parallel=True)
+def calculate_res_sparse(
+    indptr,
+    index,
+    data,
+    sums_genes,
+    sums_cells,
+    residuals,
+    sum_total,
+    clip,
+    theta,
+    n_genes,
+    n_cells,
+):
```

Comment:
```suggestion
def calculate_res_sparse(
    indptr,
    index,
    data,
    *,
    sums_genes,
    sums_cells,
    sum_total,
    clip,
    theta,
    n_genes,
    n_cells,
) -> NDArray[np.float64]:
```

Reply:
I'm ok with the * but. This function updates residuals inplace it doesn't return.